examples/shouldi/tests/test_cli.py

```diff
@@ -20,3 +20,18 @@ async def test_install(self):
             output = stdout.getvalue()
         self.assertIn("shouldi is okay to install", output)
         self.assertIn("Do not install insecure-package!", output)
+
+    async def test_install_js(self):
+        import pathlib
```

Comment:
Please clean up these testcases and make sure they are really checking the results

Reply:
You may need to modify cli.py

Reply:
Sorry , I didn't quite get it. 
On running -
`$ cd examples/shouldi
$ python setup.py test -s tests.test_cli.TestCLI.test_install_js`
It shows the accurate results.